tests/tests.rs

```diff
@@ -68,8 +72,28 @@ fn cmd_export_csv() {
     testdir.expect_path("empty_csv/cards.csv");
     testdir.expect_path("empty_csv/empty_00063_496.jpg");
     testdir.expect_path("empty_csv/empty_00060_828-00066_164.es.mp3");
+    testdir.expect_contains("empty_csv/cards.csv", FIRST_LINE);
+    testdir.expect_contains("empty_csv/cards.csv", LAST_LINE);
+}
+
+#[test]
+fn cmd_export_csv_limit() {
+    let testdir = TestDir::new("substudy", "cmd_export_csv_limit");
+    let output = testdir.cmd()
+        .args(&["export", "csv", "--limit=65"])
+        .arg(testdir.src_path("fixtures/empty.mp4"))
+        .arg(testdir.src_path("fixtures/sample.es.srt"))
+        .arg(testdir.src_path("fixtures/sample.en.srt"))
+        .output()
+        .expect("could not run substudy");
+    assert!(output.status.success());
+    testdir.expect_path("empty_csv/cards.csv");
+    testdir.expect_path("empty_csv/empty_00063_496.jpg");
+    testdir.expect_path("empty_csv/empty_00060_828-00066_164.es.mp3");
+    testdir.expect_file_contents("empty_csv/cards.csv", FIRST_LINE);
```

Comment:
We need the corresponding negative tests here, both `expect_not_path` and `expect_not_file_contents`, or something like that. Please feel free to submit a PR against `cli_test_dir` with the APIs you need!